app/components/AudioPlayer/index.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useRef } from 'react';
+import { PlayCircleFilled, PauseCircleFilled } from '@ant-design/icons';
+import { Tooltip } from 'antd';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import { compose } from 'redux';
+import If from '@components/If';
+
+const PlayIcon = styled(PlayCircleFilled)`
+  && {
+    font-size: 250%;
```

Comment:
please use fonts.js, use em for font-size as far as possible

Reply:
i missed this file , will take care of it